src/charm.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+
+import sys
+sys.path.append('lib')
+
+from ops.charm import CharmBase
+from ops.main import main
+from ops.model import ActiveStatus, MaintenanceStatus
+from oci_image import OCIImageResource, ResourceError
+
+
+class MultusCharm(CharmBase):
+    def __init__(self, *args):
+        super().__init__(*args)
```

Comment:
I think using `*args` here is a bit confusing.  It would be better to explicitly call out the `framework` and `key` params instead.

Reply:
done